KopernicusExamples/Creating New Bodies/GasGiant/GasPlanet.cfg

```diff
@@ -1,9 +1,24 @@
-@Kopernicus:AFTER[Kopernicus]
+@Kopernicus:AFTER[Kopernicus]:FOR[Bastion]
 {
+	useOnDemand = true
```

Comment:
On thing I am wondering about: Who started the trend to put useless config options into the planet configs? That value is the default, and has been the default forever. All it does is add 10 or 20 `useOnDemand = True` entries to the node which then look stupid in the ModuleManager cache.

I don't really care about that in planet packs but it shouldn't be in the examples.

Reply:
Oh ok.

Reply:
There. Better?

Reply:
yeah